formats/email_alert_signup/publisher/details.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "tags"
+  ],
+  "properties": {
+     "tags": {
+       "type": "object"
+    },
+    "breadcrumbs": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "link",
+          "title"
+        ],
+        "properties": {
+          "link": {
+            "type": "string",
+            "format": "uri"
+          },
+          "title": {
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9-]+$"
```

Comment:
The tests are falling because I forgot to remove a regex that used extended pattern matchers which the validator doesn't support. Fair point about removing this though. I've amended the commit.